book/src/intro.md

```diff
@@ -81,6 +82,20 @@ Those principles are:
 
 [up]: https://en.wikipedia.org/wiki/Unix_philosophy
 
+But with all the [great "blocks" in the Rust gamedev ecosystem][awg], why not glue your own engine together?
+
+Glueing blocks together involves a lot more than one might think. While Amethyst is quite modular by itself, we have
+a big advantage over standalone libraries: All of our crates have a common basis, [Specs][specs] (an ECS framework).
+Due to Specs' nature we have a low-cost abstraction that needs close to no glue between the crates. If another crate
+wants to add new features, it can just take the existing components and transform them to other components, so there's
+often no glue required.
+
+Also, with random crates glued together, you have to make tools yourself. With Amethyst we can combine efforts and work
+on powerful tools together.
```

Comment:
More text needed here.

Reply:
There is quite a fatal flaw in what you've written so far in that it doesn't take into account that you can use Specs by itself. From experience, I can say that the two best things with Amethyst is that it already has tons of things you may want to use already inside it and it also is opinionated on how you start your project which leads to it being more organized. That was something I certainly struggled with when making my own Specs based engine.

Reply:
I'd also add that Amethyst is doing research work too. We create new stuff and features that are just not available by themselves (assets, parallelized scripting, etc), because they need to depend on the engine itself.